### ISSUE TITLE
Analyze how to use our evm-connector with foundry toolkit

### ISSUE THREAD

**Issue by koonopek** [redstone-finance/redstone-oracles-monorepo#493]
Requested here: https://discord.com/channels/786251205008949258/1001502963975598080/1084878261756117032

**Comment by koonopek** [comment]
I have a problem with running forge build on mac M1.

```bash
> forge build
[⠊] Compiling...
[⠆] Installing solc version 0.8.19
[⠑] Successfully installed solc 0.8.19
Error: 
Solc Error: dyld: Symbol not found: __ZNSt3__113basic_filebufIcNS_11char_traitsIcEEEC1Ev
  Referenced from: /Users/michalkonopka/.svm/0.8.19/solc-0.8.19 (which was built for Mac OS X 13.0)
  Expected in: /usr/lib/libc++.1.dylib
```

Updating the whole OS should help ... : https://github.com/foundry-rs/foundry/issues/2721

**Comment by koonopek** [comment]
Solved by upgrading my OS to 13.3
